javascript/javascript1/week1/homework-w1/script.js

```diff
@@ -0,0 +1,39 @@
+//age-ify task
+let yearOfBirth = 1982;
+let yearFuture = 2025;
+age = yearFuture - yearOfBirth;
+
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//dog year task
+let dogYearOfBirth = 2000;
+let dogYearFuture = 2030;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = false;
+
+if(shouldShowResultInDogYears == false){
```

Comment:
shouldShowResultInDogYears is boolean, so you can write it like if(!shouldShowResultInDogYears)
and what happen if we want to show result in dog years?